central/declarativeconfig/manager_impl.go

```diff
@@ -224,19 +228,37 @@ func (m *managerImpl) runReconciliation() {
 	m.transformedMessagesMutex.RLock()
 	transformedMessagesByHandler := maputil.ShallowClone(m.transformedMessagesByHandler)
 	m.transformedMessagesMutex.RUnlock()
+
 	m.reconcileTransformedMessages(transformedMessagesByHandler)
 }
 
 func (m *managerImpl) reconcileTransformedMessages(transformedMessagesByHandler map[string]protoMessagesByType) {
 	log.Debugf("Run reconciliation for the next handlers: %v", maputil.Keys(transformedMessagesByHandler))
+	hasChanges, hash := calculateHashAndIndicateChanges(transformedMessagesByHandler, m.lastReconciliationHash)
+	m.lastReconciliationHash = hash
```

Comment:
What about little refactoring
```suggestion
	hasChanges := m.updateHash(transformedMessagesByHandler)
```